lib/Backend/NativeCodeGenerator.cpp

```diff
@@ -1456,7 +1458,7 @@ NativeCodeGenerator::Prioritize(JsUtil::Job *const job, const bool forceAddJobTo
 
 ExecutionMode NativeCodeGenerator::PrejitJitMode(Js::FunctionBody *const functionBody)
 {
-    Assert(IS_PREJIT_ON());
+    Assert(IS_PREJIT_ON() || functionBody->IsAsmJSModule());
```

Comment:
I think it should be AsmJsMode, not module (IIRC module flag is only set for the closure itself).